build.sh

```diff
@@ -34,6 +34,10 @@ function set_enhanced_kernel_env() {
 }
 
 function prepare() {
+    if [ -z "$SKIP_DEPENDENCIES" ]; then
+        sudo apt-get update
+        sudo apt-get install -y protobuf-compiler libbpf-dev
```

Comment:
This is not right. 
Just remember i have added a env in https://github.com/kmesh-net/kmesh/pull/854

Reply:
Hey @hzxuzhonghu. Sorry for deley reply. I was not well hence I was not able to contribute. I have made some changes please review it

Reply:
Initially, all test cases were passing, except for the DCO check, which was failing because I hadn't used --signoff. So, I decided to add --signoff and open the pull request again. However, now an additional test case has started failing.

Reply:
Hope you are well now.